tools/python/transit/transit_graph_generator.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env python2.7
+# Generates transit graph for MWM transit section generator.
+# Also shows preview of transit scheme lines.
+import argparse
+import json
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+import os.path
+import sys, io
+
+import bezier_curves
+import transit_color_palette
+
+
+class OsmIdCode:
+    NODE = 0x4000000000000000
+    WAY = 0x8000000000000000
+    RELATION = 0xC000000000000000
+    RESET = ~(NODE | WAY | RELATION)
+
+
+def get_extended_osm_id(osm_id, type):
```

Comment:
It is not a very good idea to use builtin func name as a var names

Reply:
fixed